### PR TITLE
test freetype2 flags

### PR DESCRIPTION
**Pull request by miranov25**

### CODE REVIEW

**Comment by miranov25** [comment]
replacing https://github.com/alisw/alidist/pull/2210

**Comment by miranov25** [comment]
+test

**Comment by miranov25** [comment]
I do not understand error message in test build/O2Suite/alidist_macOS
Could somebody check it?

File: xdevel.sh [hunk 1]
```diff
@@ -6,6 +6,6 @@ system_requirement_missing: |
    * On Ubuntu-compatible systems you probably need: libxpm-dev libxext-dev libx11-dev libxft-dev
 system_requirement: ".*"
 system_requirement_check: |
-  printf "#include <X11/Xlib.h>\n#include <X11/xpm.h>\n#include <X11/Xft/Xft.h>\n#include <X11/extensions/Xext.h>\n" | cc -xc - -I/opt/X11/include `pkg-config freetype2 --cflags` -c -o /dev/null
+  printf "#include <X11/Xlib.h>\n#include <X11/xpm.h>\n#include <X11/Xft/Xft.h>\n#include <X11/extensions/Xext.h>\n" | cc -xc - -I/opt/X11/include $(freetype-config --cflags 2> /dev/null && pkg-config freetype2 --cflags 2> /dev/null)  -c -o /dev/null
```
Comment (pzhristov) [inline xdevel.sh:9]: $(freetype-config --cflags 2> /dev/null && pkg-config freetype2 --cflags 2> /dev/null)
probably should be
$(freetype-config --cflags 2> /dev/null || pkg-config freetype2 --cflags 2> /dev/null)

**Comment by pzhristov** [comment]
@miranov25 The problem is not caused by this pull request, but please check my comment concerning the replacement of && by ||.

**Comment by miranov25** [comment]
Yes. I comitted fix

**Comment by miranov25** [comment]
Hello @pzhristov 
The same probelm as before - i think not related to the commit. Will you check?
Marian

**Comment by pzhristov** [comment]
@miranov25 Yes, there is one test in O2 that fails with segmentation violation (test_Framework_benchmark_ASoAHelpers), but this is not related to the pull request. Merging.

**Pull request merged**